site/SiteFooter.tsx

```diff
@@ -14,267 +14,173 @@ interface SiteFooterProps {
-                            </p>
-                            <p className="legal--last-paragraph">
-                                <span>
-                                    Our World in Data is a project of the{" "}
-                                    <a href="https://global-change-data-lab.org/">
-                                        Global Change Data Lab
-                                    </a>
-                                    , a registered charity in England and Wales
-                                    (Charity Number 1186433).
-                                </span>
-                                <a
-                                    href="https://global-change-data-lab.org/"
-                                    className="partner-logo gcdl-logo"
-                                    data-track-note="footer_navigation"
-                                >
-                                    <picture>
-                                        <source
-                                            type="image/webp"
-                                            srcSet={`${props.baseUrl}/gcdl-logo.webp`}
-                                        />
-                                        <img
-                                            src={`${props.baseUrl}/gcdl-logo.png`}
-                                            alt="Global Change Data Lab logo"
-                                            loading="lazy"
-                                            width={106}
-                                            height={127}
-                                        />
-                                    </picture>
-                                </a>
-                            </p>
-                        </div>
+        <footer className="site-footer grid grid-cols-12-full-width">
+            <div className="footer-left span-cols-5 span-sm-cols-12 col-start-2 col-sm-start-2">
+                <p className="body-3-medium">
+                    Our World in Data is a project of{" "}
+                    <a href="https://global-change-data-lab.org/">
+                        Global Change Data Lab
+                    </a>
+                    , a nonprofit based in the UK (Reg. Charity No. 1186433).
+                    Our charts, articles, and data are licensed under{" "}
+                    <a href="https://creativecommons.org/licenses/by/4.0/">
+                        CC BY
+                    </a>
+                    , unless stated otherwise. Tools and software we develop are
+                    open source under the{" "}
+                    <a href="https://github.com/owid/owid-grapher/blob/master/LICENSE.md">
+                        MIT license
+                    </a>
+                    . Third-party materials, including some charts and data, are
+                    subject to third-party licenses. See our{" "}
+                    <a href={`${props.baseUrl}/faqs`}>FAQs</a> for more details.
```

Comment:
Is using `props.baseUrl` here and below really necessary? We don't use it in other places, so if it's doing something here, we might need to add it also elsewhere (not in this PR).

Reply:
@ikesau I see you removed `baseUrl` from this link, but there are a bunch of other uses as well. Not sure, if you wanted to keep those or not.

Reply:
Oh, true. The whole `baseUrl` thing is a bit mysterious to me, so I didn't want to remove it everywhere before thinking more deeply about why it was there in the first place (instead I just tested this one FAQ link, which doesn't really make sense but whatever 😅)

I think we should just [remove all the instances in one fell swoop, though](https://github.com/owid/owid-grapher/issues/4524), just in case there somehow is a reason it needs to be this way - I'd prefer not to have this PR tied to it.

Reply:
That makes sense. Thanks for creating the issue!